src/hist.jl

```diff
@@ -207,12 +207,12 @@ function bkfe(gcounts, drv, bandwidth, range_x)
     ## Obtain kernel weights
 
     tau = 4 + drv
-    L = min(floor(tau*h/delta), M)
+    L = min(floor(Int, tau*h/delta), M)
```

Comment:
```suggestion
    L = min(Int(fld(tau*h, delta)), M)
```

Reply:
Why is that better? I've commited your suggestion but don't really understand it.